org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/util/ResourceLoadHelper.java

```diff
@@ -16,7 +16,9 @@
  * Abstraction for other test utilities that want to load a resource for a
  * textual model.
  * @author Sebastian Zarnekow - Initial contribution and API
+ * @deprecated Use org.eclipse.xtext.ui.testing.util.ResourceLoadHelper instead
```

Comment:
am not sure if we want this.
there are gazillion of our tests that use this one.
@szarnekow  what do you think.

Reply:
I found only a couple of places where this interface is used:
![image](https://user-images.githubusercontent.com/20393472/80941957-f96d3d80-8de3-11ea-9dd9-2dde11647606.png)

Reply:
these classes have gazillions of subclasses